db/schema.rb

```diff
@@ -330,7 +329,7 @@
     t.jsonb    "preferences",         default: {}
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "roles",               default: [], null: false, array: true
+    t.jsonb    "activity_count",      default: {}
```

Comment:
Ok - i'll dig into the zoo home data more tomorrow and see if it's the data is namespaced to the single workflow / project level and fix it.